gems/webrick/CVE-2017-10784.yml

```diff
@@ -0,0 +1,39 @@
+---
+gem: webrick
+cve: 2017-10784
+ghsa: 369m-2gv6-mw28
+url: https://access.redhat.com/errata/RHSA-2017:3485
+title: WEBrick RCE Vulnerability
+date: 2022-05-14
+description: |
+  The Basic authentication code in WEBrick library in Ruby before 2.2.8,
+  2.3.x before 2.3.5, and 2.4.x through 2.4.1 allows remote attackers to inject terminal
+  emulator escape sequences into its log and possibly execute arbitrary commands via
+  a crafted user name.
+cvss_v2: 9.3
+cvss_v3: 8.8
+patched_versions:
+  - ">= 1.4.0"
```

Comment:
Can you confirm that [webrick-1.4.0](https://github.com/ruby/webrick/releases/tag/v1.4.0) patches this?

Reply:
The commit url - https://github.com/ruby/webrick/commit/4ac0f3843ab82d1c31e1cfc719409208adef7813 (range 1.4.0 to 1.8.0) - includes a another commit - https://github.com/ruby/ruby/commit/6617c41292 - which mentions https://hackerone.com/reports/223363 which mentions the CVE.